<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit!</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>&lt;ADC&gt; vs Channel</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot2" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>511</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ClusterSize</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot3" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>521</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Cluster ADCs</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>541</width>
        <height>361</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>T&lt;10</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot4" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>481</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>10&lt;T&lt;20</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot5" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>481</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>20&lt;T&lt;30</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot6" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>481</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cluster ADC Counts in time bins [ns]</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Beam Prof.</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot7" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>531</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>&lt;Cluster ADC&gt; vs TDC</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot8" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>501</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>ADC vs Channel</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>521</width>
        <height>378</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Plots</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest_plot">
    <property name="title">
     <string>The 40 Thieves</string>
    </property>
   </widget>
   <addaction name="menuTest_plot"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
